sql/4_2_2-to-4_3_1_upgrade.sql

```diff
@@ -609,4 +609,8 @@ INSERT INTO `list_options` (`list_id`,`option_id`,`title`,`notes`,`seq`) VALUES
 
 #EndIf
 
-
+#IfNotRow2D list_options list_id lists option_id PageValidation
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('lists', 'PageValidation', 'PageValidation', 298, 1, 0, '', '', '', 0, 0, 1, ''),
```

Comment:
Don't include caps in the option_id (ie. page_validation or pagevalidation).
For the title, make it human readable (ie. "Page Validation").
No reason to include flags/stuff that are simply being set to default. See other inserts into list_options to see example.